src/components/AksaraBox/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Image from "next/image";
+import Text from "src/components/Text";
+
+interface OwnProps {
+  bgImage?: string;
+  title: string;
+  description: string;
+  hyperlink: string;
+}
+
+const AksaraBox: React.FC<OwnProps> = ({
```

Comment:
Can rename to something like `ContentPreview`

Reply:
done